theseus/labs/lie/functional/se3_impl.py

```diff
@@ -21,85 +21,68 @@
 
 def check_group_tensor(tensor: torch.Tensor):
     def _impl(t_: torch.Tensor):
-        if t_.ndim != 3 or t_.shape[1:] != (3, 4):
-            raise ValueError(
-                f"SE3 data tensors can only be 3x4 matrices, but got shape {t_.shape}."
-            )
         SO3.check_group_tensor(t_[:, :, :3])
 
+    if tensor.shape[-2:] != (3, 4):
+        raise ValueError(
+            f"SE3 data tensors can only be 3x4 matrices, but got shape {tensor.shape}."
```

Comment:
Now that we support multiple dimensions, I think a better message for all of these will be
```python
f"SE3 data tensors can must have shape (..., 3, 4) but got shape {tensor.shape}."
```